plugins/org.python.pydev/src/org/python/pydev/ui/pythonpathconf/package_manager/CondaPackageManager.java

```diff
@@ -35,6 +37,21 @@ public CondaPackageManager(IInterpreterInfo interpreterInfo, File prefix) {
         this.prefix = prefix;
     }
 
+    public static List<String> listCondaEnvironments(File condaExecutable) {
+        List<String> lst = new ArrayList<>();
+        String encoding = Charset.defaultCharset().displayName(); // use system encoding
+        Tuple<String, String> output = new SimpleRunner().runAndGetOutput(
+                new String[] { condaExecutable.toString(), "env", "list", "--json" }, null, null,
+                null,
+                encoding);
+        JsonObject jsonOutput = JsonValue.readFrom(output.o1).asObject();
+        JsonArray envs = jsonOutput.get("envs").asArray();
+        for (JsonValue env : envs.values()) {
+            lst.add(new Path(env.asString()).toOSString());
```

Comment:
I think you should return a `List<File>` here, so, this'd be `lst.add(new File(env.asString()))`.